apps/client/src/pages/LocationPage.js

```diff
@@ -17,12 +17,14 @@ import Error from "../components/Error";
 
 const LocationPage = ({ topic }) => {
   const { trackEvent } = useMatomo();
-  const context = useContext(Context);
+  const sessionContext = useContext(SessionContext);
+  const checkerContext = useContext(CheckerContext);
   const history = useHistory();
   const [address, setAddress] = useState(null);
   const [errorMessage, setErrorMessage] = useState();
   const { clearError, errors, register, unregister, handleSubmit } = useForm();
   const { slug, text } = topic;
+  const sessionAddress = sessionContext.address?.[slug];
```

Comment:
Again, not sure. Also question mark again.

Reply:
This questionmark needs to be there otherwise you get a `TypeError: Cannot read property 'dakkapel-plaatsen' of null` when you open a Question Page in an incognito window. And I don't want to add all slugs to the default. Marked as resolved for now.